bandyer-android-design/src/main/res/values/dialog_whiteboard.xml

```diff
@@ -40,7 +40,7 @@
         <item name="bandyer_rootLayoutStyle">@style/BandyerSDKDesign.BottomSheetDialog.Whiteboard.RootLayout</item>
         <item name="bandyer_progressStyle">@style/BandyerSDKDesign.BottomSheetDialog.Whiteboard.LoadingProgressIndicator</item>
```

Comment:
the style linked in bandyer_progressStyle has not been refactored on line 32 and 41